src/Tribe/Template.php

```diff
@@ -519,19 +533,32 @@ public function template( $name, $context = array(), $echo = true ) {
 		// Setup the Hook name
 		$hook_name = implode( '/', $namespace );
 
-		// Check if the file exists
-		$file = $this->get_template_file( $name );
+		// Cache file location and existence.
+		if ( ! isset( $file_exists[ $cache_name_key ] ) || ! isset( $files[ $cache_name_key ] ) ) {
+			// Check if the file exists
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: a4283aee6ae2f5934a98d1102998e49d</sub>